cli/tauri.js/bin/tauri.js

```diff
@@ -66,6 +71,12 @@ const tauri = function (command) {
     console.log(`Invalid command ${command}. Use one of ${cmds.join(', ')}.`)
   }
 }
+// notifying updates.
+const pkg = JSON.parse(fs.readFileSync(__dirname + '/../package.json')); 
+if (pkg.version.indexOf('0.0.0') !== 0 && noUpdates !== true) {
+  require('update-notifier')({ pkg }).notify();
```

Comment:
This isn't specified in the `package.json`.

Reply:
I've added all the dependencies, as well as committed all the changes here's how the cli looks ->
![Screen Shot 2021-03-26 at 9 39 33 PM](https://user-images.githubusercontent.com/72539206/112709068-c6a8b980-8e7b-11eb-93fc-018a1353aa72.png)